EveripediaNetwork/ep-ui
# Revamped Explorer list on wiki details

# Revamped Explorer list on wiki details

## How should this be tested?

1. Go to wiki details page
2. Click on "Edit Wiki Details" from the right panel
3. Check Links and Link Wikis dropdown

## Notes or observations

_Also replaced Link Wikis select component with the new one_

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/2906
